solid/lib/Controller/ProfileController.php

```diff
@@ -274,11 +274,13 @@ private function getUserProfile($userId) {
 			$addressBooks = $this->contactsManager->getUserAddressBooks();
 			$friends = [];
 			foreach($addressBooks as $k => $v) {
-			  $results = $addressBooks[$k]->search('', ['FN'], ['types' => true]);
-			  foreach($results as $found) {
-				  foreach($found['URL'] as $i => $obj) {
-					array_push($friends, $obj['value']);
-				  }
+				$results = $addressBooks[$k]->search('', ['FN'], ['types' => true]);
+				foreach($results as $found) {
+					if (isset($found['URL']) && is_array($found['URL'])) {
```

Comment:
Currently this breaks the test-suite run. We might need to add more setup so that the URL is available within the addressbook in order for the tests to pass.